components/ShadowElevations/src/MDCShadowElevations.h

```diff
@@ -18,90 +18,85 @@
 #import <CoreGraphics/CoreGraphics.h>
 #import <Foundation/Foundation.h>
 
-// This macro is introduced in Xcode 9.
-#ifndef CF_TYPED_ENUM // What follows is backwards compat for Xcode 8 and below.
-#if __has_attribute(swift_wrapper)
-#define CF_TYPED_ENUM __attribute__((swift_wrapper(enum)))
+#ifdef NS_TYPED_EXTENSIBLE_ENUM // This macro is introduced in Xcode 9.
+#define MDC_SHADOW_ELEVATION_TYPED_EXTENSIBLE_ENUM NS_TYPED_EXTENSIBLE_ENUM
+#elif defined(CF_TYPED_EXTENSIBLE_ENUM) // What follows is backwards compat for Xcode 8 and below.
```

Comment:
Can you verify that these comments are accurate now? I think they need to be updated.